### PR TITLE
Setting up Front End

### PR DESCRIPTION
**Pull request by Georges1998**
# Description
Setting up the front end

# Checklist:
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

**Comment by Georges1998** [comment]
Setting up Front-End

File: frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# IntelliJ related
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# The .vscode folder contains launch configuration and tasks you configure in
+# VS Code which you may wish to be included in version control, so this line
+# is commented out by default.
+#.vscode/
```
Comment (yuedai0531) [inline frontend/.gitignore]: un-comment this or remove

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+These instructions will get you up and running for the mobile front-end development on your local machine.
+
+### Flutter Installation
+
+This installation follows these outlines.
+```
+https://flutter.dev/docs/get-started/install/macos
+https://flutter.dev/docs/get-started/install/windows
+
+https://www.youtube.com/watch?v=hL7pkX1Pfko
```
Comment (yuedai0531) [inline frontend/README.md]: nice add :)

**Comment by Georges1998** [comment]
I just realized that the **lib** folder which contains the code did not get pushed to git. The folder is greyed out and git is not tracking it. Anyone knows how to solve this? 

image

**Comment by Ty-Won** [comment]
@Georges1998 hey! you can have different gitignores for each directory so depending on what you specify will determine what gets ignored. Perhaps lib is not added cause it might not be tracked? Is this new and have you made a commit with it before? if not add it, commit it and it should be fine. You can make a commit to the same branch and this PR will update

**Comment by Georges1998** [comment]
@Ty-Won thanks! @yuedai0531 I fixed some stuff and added the authentication page code. Everything is in this pull request is that okay?

**Pull request merged**